css/header.css

```diff
@@ -1,6 +1,6 @@
 .sub-header {
     background: var(--primary);
-    color: var(--white);
+    color: var(--white-color);
     padding: 0.3rem;
```

Comment:
if the value is less than 1rem you can use px like 3px

Reply:
Updated it. Thanks